shared/actions/platform-specific.native.js

```diff
@@ -274,6 +274,13 @@ function openAppSettings() {
   Linking.openURL('app-settings:')
 }
 
+const getMimeTypeFromURL = (url: string): Promise<string> =>
+  new Promise((resolve, reject) => {
+    fetch(url, {method: 'HEAD'}) // eslint-disable-line no-undef
+      .then(response => resolve(response.headers.get('Content-Type')))
+      .catch(reject)
```

Comment:
Why is this lint disable needed

Reply:
I think fetch is injected by react native or something. The linter was complaining about fetch not defined.